### PR TITLE
Switching from classic fetch to using the npm_f1mv_api module

### PR DESCRIPTION
**Pull request by LapsTimeOFF**
Switching from classic fetch to using the npm_f1mv_api module

### CODE REVIEW

**Review by TheBaconCat — APPROVED** [review]
Looks Good to Me

**Comment by JJWatMyself** [comment]
I think I understand what I'm looking at. I have not pulled to test though.

**Review by JJWatMyself — APPROVED** [review]

File: renderer.ts [hunk 1]
```diff
@@ -725,15 +823,15 @@
         <path d="M2 12h20M12 2a15.3 15.3 0 0 1 4 10 15.3 15.3 0 0 1-4 10 15.3 15.3 0 0 1-4-10 15.3 15.3 0 0 1 4-10z"/>
       </svg>`);
         $('#networkSettings').append('<h5>Network</h5>');
-        $('#networkSettings').append('<p>IP : </p>');
+        $('#networkSettings').append('<span>IP : </span>');
         $('#networkSettings').append(
-            `<input type="text" class="form-control text-bg-dark" placeholder="localhost" value=${host} id="ip">`
+            `<input type="text" class="form-control text-bg-dark" placeholder="localhost" value="${config.host}" id="ip">`
```
[the remark this answers is not in the document]
Comment (LapsTimeOFF) [reply renderer.ts:828]: Dismissed

[the remark this answers is not in the document]
Comment (TheBaconCat) [reply renderer.ts:828]: Sounds Good

**Review by TheBaconCat — APPROVED** [review]

**Review by TheBaconCat — APPROVED** [review]

**Pull request merged**